tests/ga/test_monitor.py

```diff
@@ -98,16 +105,21 @@ def test_it_should_invoke_all_periodic_operations(self):
             def mock_run(self):
                 invoked_operations.append(self._name)
 
-            with patch.object(PeriodicOperation, "run", side_effect=mock_run, spec=MonitorHandler.run):
+            with patch.object(PeriodicOperation, "run", side_effect=mock_run, autospec=True):
                 monitor_handler.run_and_wait()
 
                 expected_operations = [
-                    "reset_loggers", "collect_and_send_events", "send_telemetry_heartbeat",
-                    "poll_telemetry_metrics usage", "send_telemetry_metrics usage", "send_host_plugin_heartbeat",
-                    "send_imds_heartbeat", "log_altered_network_configuration"
+                    'poll resource usage',
```

Comment:
the names (and number) of the operations have changed over time and were not updated because the test was never reporting the failures